js/article-view.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
```

Comment:
For view logic, the concept of 'self-documenting code' is usually a bit different. Which is easier to read:

```
function view(title) {
  return "<div class='my-view'><h1>" + title + "</h1></div>"
}
```

or

```
function getTitle(title) {
  return "<h1>" + title + "</h1>";
}

function getView() {
  var view = "<div class='my-view'>" + getTitle(myTitle) + "</div>";
}
```
Essentially, when you are writing a view, the view is it's own documentation, so I tend to think it's cleaner to keep the chunks larger, and only break out meaningful, reusable components (e.g. a menu or a form) into subviews